app/services/void_participant_declaration.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+class VoidParticipantDeclaration
+  attr_accessor :cpd_lead_provider, :id
+
+  def call
+    declaration = ParticipantDeclaration.find(id)
+
+    raise ActionController::BadRequest, "Declaration is already voided" if declaration.voided
+
+    latest_declaration = declaration.participant_profile.participant_declarations.order(declaration_date: :desc).first
+    raise ActionController::BadRequest, "Can only void last declaration" if latest_declaration != declaration
```

Comment:
Why?

Reply:
That's what the ticket says 😂 

I think it's unclear how we would handle voiding if there is a follow-up declaration. Presumably if the started declaration is voided, then retained declaration is at least suspect too. It's a business problem to be tackled lated, so we just don't lt the providers do it for now.

Reply:
@kicferk1 What if they got the date wrong on the `started` declaration, but only realised later, after the first retained one?

Reply:
I think if that's the case the voiding will have to trigger a clawback, because we will have most likely processed it. When we build that functionality we will probably add in the ability to void more declarations.

Or keep the scenario that you described as a support request until it becomes too commonplace and annoying.